### PR TITLE
update recipe with version 2.2.6

### PR DESCRIPTION
**Pull request by RajeshwariSubramaniam**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,13 @@
-{% set version = "3.0.9" %}
+{% set name = "openpyxl" %}
+{% set version = "2.2.6" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml:2]: This is not an update. We can try a one-time PR to a branch to create an older release but without some context on badly this is needed it is probably not worth the effort.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -15,31 +16,28 @@ build:
 
 requirements:
   host:
-    - python >=3.6
```
Comment (ocefpaf) [inline recipe/meta.yaml:18]: While 2.6  may be accurate we are only building >=3.6 in conda-forge for a while now. We are in fact about to drop 3.6 soon.

Comment (RajeeGit) [reply recipe/meta.yaml:18]: Does this mean, the requested package cannot be built?

**Comment by ocefpaf** [comment]
Closing this as stale. If you need the old package we can try to put it in a branch,

**Pull request closed without merging**